src/iso.rs

```diff
@@ -91,6 +209,35 @@ pub fn iso_show(config: &IsoShowConfig) -> Result<()> {
     Ok(())
 }
 
+pub fn iso_show_kargs(config: &IsoShowKargsConfig) -> Result<()> {
+    let mut file = OpenOptions::new()
+        .read(true)
+        .open(&config.input)
+        .chain_err(|| format!("opening {}", &config.input))?;
+
+    let (area_size, area_offsets) = find_karg_embed_areas(&mut file)?;
+    // we know there's at least one; checked in find_karg_embed_area()
+    file.seek(SeekFrom::Start(area_offsets[0]))
+        .chain_err(|| "seeking to karg area offset")?;
+
+    let mut area = [0u8; COREOS_KARG_EMBED_AREA_MAX_SIZE];
+    file.read_exact(&mut area[..area_size])
+        .chain_err(|| "reading karg embed area")?;
+
+    let s = std::str::from_utf8(&area[..area_size]).chain_err(|| "invalid UTF-8 area")?;
+    let s = s.trim_end_matches('#');
+    if s.is_empty() {
+        bail!("No embedded additional kernel args.");
```

Comment:
Is that an error?

Reply:
This matches the semantics for `iso ignition show`. :)